code/modules/organs/internal/_internal.dm

```diff
@@ -137,43 +236,143 @@
+		var/is_robotic = BP_IS_ROBOTIC(src) || BP_IS_ASSISTED(src)
+		var/is_organic = BP_IS_ORGANIC(src) || BP_IS_ASSISTED(src)
+		var/list/possible_wounds = list()
+
+		if(is_organic)
+			LAZYADD(possible_wounds, typesof(/datum/component/internal_wound/organic/bone_fracture))
+		if(is_robotic)
+			LAZYADD(possible_wounds, typesof(/datum/component/internal_wound/robotic/deformation))
+
+		if(LAZYLEN(possible_wounds))
+			var/choice = pick(possible_wounds)
+			add_wound(choice)
+
+		if(owner)
+			owner.visible_message(
+				SPAN_DANGER("You hear a loud cracking sound coming from \the [owner]."),
+				SPAN_DANGER("Something feels like it shattered in your [name]"),
+				SPAN_DANGER("You hear a sickening crack.")
+			)
+			if(owner.species && !(owner.species.flags & NO_PAIN))
+				owner.emote("scream")
 
-// Gets a list of surgically treatable conditions
-/obj/item/organ/internal/get_conditions()
-	var/list/conditions_list = ..()
-	var/list/condition
+			// Fractures have a chance of getting you out of restraints
+			if(prob(25))
+				parent.release_restraints()
 
-	if(damage > 0)
+/obj/item/organ/internal/proc/mend()
+	if(LAZYACCESS(organ_efficiency, OP_BONE))
+		parent.status &= ~ORGAN_SPLINTED
+
+		// Determine possible wounds based on nature and damage type
+		var/is_robotic = BP_IS_ROBOTIC(src) || BP_IS_ASSISTED(src)
+		var/is_organic = BP_IS_ORGANIC(src) || BP_IS_ASSISTED(src)
+
+		if(is_organic)
+			remove_wound(/datum/component/internal_wound/organic/bone_fracture)
+		if(is_robotic)
+			remove_wound(/datum/component/internal_wound/robotic/deformation)
+
+/obj/item/organ/internal/get_actions()
+	var/list/actions_list = ..()
+
+	if(LAZYACCESS(organ_efficiency, OP_BONE))
 		if(BP_IS_ROBOTIC(src))
-			condition = list(
-				"name" = "Damage",
-				"fix_name" = "Repair",
-				"step" = /datum/surgery_step/robotic/fix_organ,
-				"organ" = "\ref[src]"
-			)
+			if(parent.status & ORGAN_BROKEN)
+				actions_list.Add(list(list(
+					"name" = "Mend",
+					"organ" = "\ref[src]",
+					"step" = /datum/surgery_step/robotic/fix_bone
+				)))
 		else
-			condition = list(
-				"name" = "Damage",
-				"fix_name" = "Heal",
-				"step" = /datum/surgery_step/fix_organ,
-				"organ" = "\ref[src]"
-			)
+			actions_list.Add(list(list(
+				"name" = (parent.status & ORGAN_BROKEN) ? "Mend" : "Break",
+				"organ" = "\ref[src]",
+				"step" = (parent.status & ORGAN_BROKEN) ? /datum/surgery_step/mend_bone : /datum/surgery_step/break_bone
+			)))
+			actions_list.Add(list(list(
+					"name" = "Replace",
+					"organ" = "\ref[src]",
+					"step" = /datum/surgery_step/replace_bone
+				)))
+
+	return actions_list
+// End of the bone zone
+
+// Mutations
+/obj/item/organ/internal/proc/unmutate()
+	var/is_organic = BP_IS_ORGANIC(src) || BP_IS_ASSISTED(src)
+
+	if(!is_organic)
+		return
+
+	var/list/wounds = GetComponents(/datum/component/internal_wound/organic/radiation)
+
+	for(var/datum/component/wound in wounds)
+		remove_wound(wound)
```

Comment:
Or even like that, should be enough, since we're just passing a reference.
```suggestion
	if(!BP_IS_ORGANIC(src) || !BP_IS_ASSISTED(src))
		return

	for(var/wound in GetComponents(/datum/component/internal_wound/organic/radiation))
		remove_wound(wound)
```